metrics-api/src/main/java/software/amazon/swage/metrics/ContextData.java

```diff
@@ -18,33 +18,33 @@
 import software.amazon.swage.collection.TypedMap;
 
 /**
- * Builder for data carried with the context under which a metric is reported.
+ * Builder for attributes carried with the context under which a metric is reported.
  *
  * <p>
- * Roughly equivalent to a request context, this is the metadata associated with
+ * Roughly equivalent to a request context, this is the attributes associated with
  * a metric context that allows relations and aggregations between events.
  * Things like request id, tracing id, etc. will be carried here.
  *
  * <p>
  * Built instances are immutable.  Metric recording will use
- * the data across threads as needed, and relies on the data being
+ * the attributes across threads as needed, and relies on the attributes being
  * unchanged for a particular Context.
  *
  * <p>
  * Different application domains may extend the builder to facilitate adding
- * the appropriate data.
+ * the appropriate attributes.
  *
- * A domain-specific data builder might look like:
+ * A domain-specific attributes builder might look like:
  * <pre>{@code
 public class DomainSpecificData extends ContextData {
```

Comment:
With the change from "data" to "attributes" when talking about contexts, should we rename ContextData? Or is it even necessary as a concept? it seems to just be a holder for attribute keys. Is there any reason we need to couple that to TypedMapBuilder?  We can take this outside of this PR if you want.

Reply:
I was wondering that too, but was leaving things as they are for now.